uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Form/AbsenceType.tpl

```diff
@@ -247,10 +247,7 @@
                   preferredFormat: "hex3",
                   allowEmpty:true,
                   showPaletteOnly: true,
-                  showPalette:true,
-                  {/literal}
-                  palette: {$availableColors}
-                  {literal}
```

Comment:
why did we remove this?

Reply:
Because we remove everything related to colours palette from PHP. If we leave this it would access not anymore existing variable.

The variable was removed here https://github.com/compucorp/civihr/pull/2883/files#diff-5d6ce48c2bb46daa103a184e08eaf369L56